types/coin.go

```diff
@@ -269,6 +269,23 @@ func (coins Coins) safeAdd(coinsB Coins) Coins {
 	}
 }
 
+// ContainsDenomsOf returns true if coinsB' denom set
+// is subset of the receiver's denoms.
+func (coins Coins) ContainsDenomsOf(coinsB Coins) bool {
```

Comment:
`ContainsDenomsOf` is somewhat of a misnomer here. There is no direct hint that ALL denoms must be contained -- which is the case. May I suggest `ContainsAllDenoms` or `IsDenomSubset`?

Reply:
Love `IsDenomSubset`, deal!